tests/foreman/ui/test_contentview.py

```diff
@@ -3867,3 +3867,38 @@ def test_positive_inc_publish_cv(session, module_org):
         )
         cvv = entities.ContentView(id=cv.id).read().version[1].read()
         assert cvv.package_count == 32
+
+
+@pytest.mark.tier2
+def test_positive_able_to_disable_and_enable_rhel_repos(session, target_sat):
+    """Upstream repo name changes shouldn't negatively affect a user's ability
+    to enable or disable a repo
+
+    :id: 205a1c05-2ac8-4c60-8d09-016bbcfdf538
+
+    :expectedresult: User is able to enable and disabled repos without issues
+
+    :bz: 1973329
```

Comment:
```suggestion
    :BZ: 1973329
```

Reply:
Nonblocking - It will be good to have this parameter as well `:CaseAutomation: Automated` :)